# This service is used to retrieve an existing image thumbnail from an existing
# object on the interoperability server in a compressed format.

# Object ID.
uint64 id

---

# Whether the request was successful.
bool success

# Image thumbnail.
sensor_msgs/CompressedImage image
